tests/e2e/Services/Sites/SitesCustomServerTest.php

```diff
@@ -73,12 +74,13 @@ public function testConsoleAvailabilityEndpoint(): void
             'framework' => 'other',
             'buildRuntime' => 'ssr-22',
             'outputDirectory' => './',
-            'subdomain' => 'test-site',
```

Comment:
I would expect to see this change more 🤔
Please double-check we dont use this subdomain param anywhere anymore, in tests. Both sites and functions

Reply:
Verified, I don't see any more instances